src/jit/bitsetasuint64.h

```diff
@@ -139,6 +144,11 @@ class BitSetOps</*BitSetType*/ UINT64,
         return (bs1 & bs2) == 0;
     }
 
+    static void LivenessD(Env env, UINT64& in, const UINT64 def, const UINT64 use, const UINT64 out)
```

Comment:
They were missed for BSUInt64Class, it is why I was not able to use BSUInt64Class as VARSET_REP before.